cdap-api/src/main/java/co/cask/cdap/api/common/Scope.java

```diff
@@ -25,8 +25,7 @@
 public enum Scope {
   DATASET("dataset"),
   MAPREDUCE("mapreduce"),
-  SPARK("spark"),
-  FLOWLET("flowlet");
```

Comment:
the `FLOWLET` was newly added in 3.5 for the `@Batch` change in https://github.com/caskdata/cdap/pull/6463

Hence removing is ok.